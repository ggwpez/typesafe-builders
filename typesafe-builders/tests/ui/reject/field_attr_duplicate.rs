use typesafe_builders::prelude::*;

#[derive(Builder)]
struct Struct {
	#[builder(optional)]
	#[builder(optional)]
	x: u8,
}

fn main() { }
