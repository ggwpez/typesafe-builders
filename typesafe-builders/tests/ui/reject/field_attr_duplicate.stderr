error: Duplicate attribute on struct field
 --> tests/ui/reject/field_attr_duplicate.rs:6:2
  |
6 |     #[builder(optional)]
  |     ^^^^^^^^^^^^^^^^^^^^
